contrib/seeds/generate-seeds.py

```diff
@@ -136,6 +139,8 @@ def process_nodes(g, f, structname):
             continue
 
         spec = parse_spec(line)
+        if spec == -1:  # ignore this entry (e.g. no longer supported addresses like TORV2)
```

Comment:
I would prefer using the sentinel value `None` instead of `-1` then check with `spec is None` (just a small nit though, no need to change if no other changes are necessary).

Reply:
done